cmd/podman/containers/stats.go

```diff
@@ -72,6 +73,7 @@ func statFlags(cmd *cobra.Command) {
 
 	flags.BoolVar(&statsOptions.NoReset, "no-reset", false, "Disable resetting the screen between intervals")
 	flags.BoolVar(&statsOptions.NoStream, "no-stream", false, "Disable streaming stats and only pull the first result, default setting is false")
+	flags.IntVar(&statsOptions.Interval, "interval", 1, "Time in seconds between stats reports")
```

Comment:
```suggestion
	intervalFlagName := "interval"
	flags.IntVar(&statsOptions.Interval, intervalFlagName, 1, "Time in seconds between stats reports")
	_ = cmd.RegisterFlagCompletionFunc(intervalFlagName, completion.AutocompleteNone)
```

Reply:
Should we up it by default to 5?  Also proabably should be added to containers.conf, to be set.

Reply:
@rhatdan : considered it but hesitated to do without asking here. My personal opinion: 5 sec is fine, 1 sec is really aggressive in case you have more then a handful containers.

Reply:
Lets bump it to 5 seconds.

Reply:
Ok, it's now 5 seconds for cli and api.